Source/IGListAdapter.m

```diff
@@ -1015,6 +1015,25 @@ - (void)scrollToSectionController:(IGListSectionController<IGListSectionType> *)
     [self.collectionView scrollToItemAtIndexPath:indexPath atScrollPosition:scrollPosition animated:animated];
 }
 
+- (void)invalidateLayoutForSectionController:(IGListSectionController<IGListSectionType> *)sectionController {
+    IGAssertMainThread();
+    IGParameterAssert(sectionController != nil);
+    
+    NSInteger itemCount = [sectionController numberOfItems];
```

Comment:
Removed the need for this variable by having a single API, though thanks for explaining it to me!